src/heroes_build_scrapper/data.py

```diff
@@ -73,12 +75,6 @@ def load_builds(hero):
     with open(filename, 'r') as fp:
         load_list = json.load(fp)
 
-    for i in range(0, len(load_list) - 1, 2):
```

Comment:
I'm sorry but I didn't understand what is going on here, what does the function returns then?

Reply:
Good catch!!! That's a mistake.

I rebased after your `pr` #21. That's when the mistake occurred.